pkg/testutil/remote_config_fake.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testutil provides utilities that are intended to enable easier
+// and more concise writing of unit test code.
+package testutil
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/viper"
+)
+
+type FakeRemoteConfig struct {
```

Comment:
Comment in this file. Fake usually should come with its own unit test.

Reply:
Added comment. The implementation is pretty simple, just call functions in viper directly. I think we can pass the unit test of the fake now. In addition, we currently just use this fake for UT.  Looks like in jvs, we don't write UT for test utility methods.

Reply:
A "fake" and a general test util is different. Due the simplicity in the implementation (simply calling viper), I'm ok not skipping it for now.

Reply:
Are you ok skipping the UT of fake  or not?